### ISSUE TITLE
6025: Hide legend bubbles/pies on zoom & let bubbles/pies grow together with the map on zoom

### ISSUE THREAD

**Issue by larafaelivrin** [StataBS/indikatoren#550]
I improved the chart in branch issue-325.

**Comment by jb3-2** [comment]
Hi, when you click the legend the wrong legend title disappears: 

![screen shot 2017-12-18 at 14 11 07](https://user-images.githubusercontent.com/7261044/34107701-7ecc2074-e3fd-11e7-989e-8931685907f9.png)

**Comment by jb3-2** [comment]
Tooltips on pies should also contain description of the data, like the tooltips on the Wohnviertel. Otherwise we don't know what we're looking at. Compare the tooltips in the screenshots below: 

OK: Descripbes the data ("Anzahl Zugezogene pro 100 Einwohner"): 
![screen shot 2017-12-18 at 15 54 08](https://user-images.githubusercontent.com/7261044/34112132-eaf1a3ce-e40b-11e7-9d1a-616f327430cc.png)


Not OK: Just says "Schweiz" and "Hirzbrunnen". I think it should say "Anteil Zugezogene nach Zuzugsland: Schweiz", then in the 2nd line: "Hirzbrunnen: 54,0%". 
![screen shot 2017-12-18 at 15 54 27](https://user-images.githubusercontent.com/7261044/34112124-e3722ef2-e40b-11e7-8a41-59fdf8cf1f95.png)


@larafaelivrin  @JE1982 We probably have to improve this on all mappie and mapcolumn charts. Maybe not for tomorrow, but soon thereafter. Do you agree?

**Comment by larafaelivrin** [comment]
I agree and implemented your suggestion. Thank you for your feedback. I also realized that maybe it is nicer to hide all legend items when zooming in as there are to many overlaps which make the map "noisy"(see image below). Especially, as it is possible to get all necessary legend informations via tooltip: 

![image](https://user-images.githubusercontent.com/23261816/34113298-8dd010a0-e40f-11e7-9bdd-bfce573d2a44.png)

**Comment by larafaelivrin** [comment]
Change colour: red green
